JavaScriptSPA/graphConfig.js

```diff
@@ -0,0 +1,10 @@
+  // Add here the endpoints for MS Graph API services you would like to use.
+  const graphConfig = {
```

Comment:
The Azure portal replaces the endpoint host URL with protocol+domain+trailing slash. Update this to so the URL won't break.